report_thesis/src/sections/introduction.tex

```diff
@@ -1,19 +1,20 @@
-The NASA Viking missions in the 1970s were the first to successfully land a rover on Mars, aiming to determine if life existed on the planet.
-One experiment suggested the presence of life, but the results were ambiguous and inconclusive, and NASA was unable to repeat the experiment due to budget constraints\cite{marsnasagov_vikings}.
+The NASA Viking missions in the 1970s were the first to successfully land on Mars, aiming to determine if life existed on the planet.
+One experiment suggested the presence of life, but the results were ambiguous and inconclusive, and NASA was unable to repeat the experiment.
+Nevertheless, these missions were deemed a monumental success and advanced our knowledge of the Martian environment.\cite{marsnasagov_vikings}
 
-A few decades later, the philosophy of Martian exploration had shifted from searching for life to investigating whether Mars ever had the conditions to support life as we know it.
-The \gls{mer} mission, which included the Spirit and Opportunity rovers, discovered clear evidence that water once flowed on Mars.
-As water alone is not enough to support life, NASA shifted their focus to search for organic material as well\cite{marsnasagov_observer, marsnasagov_spirit_opportunity}.
+Leveraging the knowledge gained from the Viking missions, NASA launched the \gls{mer} mission in 2003 to investigate whether Mars ever had the conditions to support life as we know it.
+The mission landed two rovers, Spirit and Opportunity, on Mars in January 2004, and they quickly discovered clear evidence that water once flowed on Mars.
+However, since water alone is not enough to support life, the next objective was to search for organic material as well.\cite{marsnasagov_observer, marsnasagov_spirit_opportunity}
 
 The Curiosity rover landed on Mars in August 2012 inside Gale Crater as part of the \gls{msl} mission with this very purpose.
-Its sophisticated equipment quickly discovered that the conditions to support life as we know it had existed on Mars through chemical and mineral evidence.\cite{marsnasagov_chemcam}
+Thanks to its sophisticated equipment, Curiosity was quickly able to find evidence of past habitable environments on Mars based on chemical and mineral findings.\cite{marsnasagov_chemcam}
 
 One of the instruments aboard the rover is the \gls{chemcam} instrument, which is a remote-sensing laser instrument used to gather \gls{libs} data from geological samples on Mars.
-\gls{libs} is a non-invasive technique that enables rapid analysis without the need for sample preparation by using a laser to ablate and remove surface contaminants to expose the underlying material and generate a plasma plume from the now-exposed sample material.
+\gls{libs} is a standoff technique that enables rapid analysis without the need for sample preparation by using a laser to ablate and remove surface contaminants to expose the underlying material and generate a plasma plume from the now-exposed sample material.
```

Comment:
Maybe isn't clear what standoff means? The technique being standoff is more related to being able to shoot and gather LIBS data at a distance, afaik.

Not sure what is meant by `without the need for sample preparation`.

Cite?

Reply:
> The technique being standoff is more related to being able to shoot and gather LIBS data at a distance, afaik.
Yes. Jens specifically suggested we use that word instead.

> Not sure what is meant by without the need for sample preparation.
No need to manually prepare the sample - LIBS does that as part of the laser removing dust.

Added a cite.

Reply:
> Yes. Jens specifically suggested we use that word instead.

I know. He also suggested `non-destructive`. 
My issue isn't with the wording, but whether it's clear to the readers of our CS paper.

> No need to manually prepare the sample - LIBS does that as part of the laser removing dust.

A benefit of using LIBS on Mars is that it removes dust from the samples, sure. But it's the lack of clarity regarding "sample preparation" I'm addressing. What would standard sample preparation be? Does it matter here?
That it removes dust is relevant - dust is a surface contaminant, after all. But what exactly is not being done, that other methods do?
I don't think it's relevant to explicitly say, so I'm voting for removing the part about sample preparation (but keeping the part about dust removal).

> Added a cite.

The cite helps for sure